topics/sequence-analysis/tutorials/quality-control/tutorial.md

```diff
@@ -289,11 +289,11 @@ FastQC produces other diagnostic plots to assess sample quality.
 {: .question}
 
 
-# Trim and filter
+# Filter and Trim
 
-Based on the information provided by the quality graphs, the quality of the sequences drops at the end of the sequences. It could cause bias in downstream analyis with these potentially wrong nucleotides.
+We will filter and trim the sequences based on the information provided by the quality graphs, the quality of the sequences drops at the end of the sequences. This could cause bias in downstream analyes with these potentially incorrectly called nucleotides.
```

Comment:
```suggestion
We will filter and trim the sequences based on the information provided by the quality graphs, the quality of the sequences drops at the end of the sequences. This could cause bias in downstream analyses with these potentially incorrectly called nucleotides.
```